docs/Makefile

```
@@ -175,3 +175,6 @@ pseudoxml:
 	$(SPHINXBUILD) -b pseudoxml $(ALLSPHINXOPTS) $(BUILDDIR)/pseudoxml
 	@echo
 	@echo "Build finished. The pseudo-XML files are in $(BUILDDIR)/pseudoxml."
+
+show:
+	@python -c "import webbrowser; webbrowser.open_new_tab('file://$(shell pwd)/_build/html/index.html')"
```

The webbrowser module has a [documented](https://docs.python.org/3/library/webbrowser.html) `-m` CLI. Hopefully I've got the quoting right here:

```suggestion
	@python -m webbrowser -t "file://$(shell pwd)/_build/html/index.html"
```